frame-metadata/src/v16.rs

```diff
@@ -187,14 +187,14 @@ impl IntoPortable for RuntimeApiMethodParamMetadata {
 pub struct ExtrinsicMetadata<T: Form = MetaForm> {
 	/// Extrinsic versions.
```

Comment:
Just a nit for clarity
 
```suggestion
	/// Extrinsic versions supported by the runtime.
```